tools/topology/topology1/sof-acp-renoir.m4

```diff
@@ -0,0 +1,66 @@
+#
+# Topology for Renoir with I2S SP and DMIC.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`acp-sp.m4')
+include(`acp-dmic.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ACP DSP configuration
+include(`platform/amd/acp.m4')
+
+# Playback pipeline 1 on PCM 0 using max 2 channels of s16le.
+# Schedule 96 frames per 2000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-passthrough-playback.m4,
```

Comment:
I strongly suggest you use separate pipelines for host and dai parts, as done e.g. for nocodec and being added for HDaudio/SoundWire. It's more flexible. Using a single pipeline all the way from host to dai was a tragic mistake done many years ago. It doesn't scale.

Even if you only have one PCM device today, having a mixer by default will help you scale.

Reply:
Thanks for the review and suggestions. This is the initial topology support. We will add more topology files and take care as suggested.